### PR TITLE
Add html CV

### PR DESCRIPTION
**Pull request by raw-leak**

### CODE REVIEW

File: staff/mykhaylo-gusak/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    </section>
+
+    <section>
+
+        <header>
+            <h2>Experiencia</h2>
+        </header>        
+        
+        <article>
+            <h3>Geodisa S.L.</h2>
```
Comment (lexbarba) [inline staff/mykhaylo-gusak/cv/index.html]: @MichaelXXI open and closing tag should be the same, please update all `h3-h2` in the document. Thanks.

**Review by lexbarba — CHANGES_REQUESTED** [review]

File: staff/mykhaylo-gusak/cv/index.html (hunk 1 above)
Comment (raw-leak) [reply staff/mykhaylo-gusak/cv/index.html]: Already done.

File: staff/mykhaylo-gusak/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    <title>CV Mykhaylo Gusak</title>
+</head>
+
+<body>
+
+    <header>
+        <h1>Currículum</h1>
+        <section>
+            <h2>Datos personales</h2>
+            <p><b>Nombre:</b>Mykhaylo Gusak</p>
```
Comment (robert-z) [inline staff/mykhaylo-gusak/cv/index.html]: @MichaelXXI  `<b>` is not a semantic tag 😄

File: staff/mykhaylo-gusak/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+    </section>
+
+    <section>
+
+        <header>
+            <h2>Idiomas:</h2>
+        </header>
+
+        <!--
```
Comment (robert-z) [inline staff/mykhaylo-gusak/cv/index.html]: @MichaelXXI remove this comment if it is not necessary

File: staff/mykhaylo-gusak/cv/index.html (hunk 2 above)
Comment (robert-z) [reply staff/mykhaylo-gusak/cv/index.html]: More info in the following [link](https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em)

Comment (raw-leak) [reply staff/mykhaylo-gusak/cv/index.html]: @robert-z Thnx , already solved.

File: staff/mykhaylo-gusak/cv/index.html (hunk 3 above)
Comment (raw-leak) [reply staff/mykhaylo-gusak/cv/index.html]: @robert-z Comment already removed.

**Pull request merged**